index.rst

```diff
@@ -41,7 +41,8 @@ visit our `gallery </docs/dev/auto_examples>`__.
 
         from skimage import data, io, filters
 
-        image = data.coins() # or any NumPy array!
+        # ... or any other NumPy array!
+        image = data.coins()
```

Comment:
@soupault I agree that we should hear from others, but my suggested fix would be to put a newline after the comment.